cpp/oneapi/dal/algo/decision_forest/test/spmd.cpp

```diff
@@ -58,13 +58,27 @@ class df_spmd_test : public df_test<TestType, df_spmd_test<TestType>> {
         const auto split_responses = te::split_table_by_rows<float_t>(this->get_policy(),
                                                                       input.get_responses(),
                                                                       split_count);
+        bool is_weighted = input.get_weights().get_row_count() > 0;
```

Comment:
I didnt found a test case with weghts_enabling, should we add it?

Reply:
These tests were failing when these changes were merged initially: https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/algo/decision_forest/test/spmd.cpp#L389-L440, so I think there is at least some coverage. May be worth adding more.